packages/react/src/components/TextEllipsis/index.tsx

```diff
@@ -7,14 +7,24 @@ import type {
   PolymorphicComponent
 } from '../../utils/polymorphicComponent';
 
-interface TextEllipsisProps
+interface TextEllipsisBaseProps
   extends PolymorphicProps<React.HTMLAttributes<HTMLElement>> {
   children: string;
   maxLines?: number;
   refProp?: string;
+  hideTooltip?: boolean;
```

Comment:
I debated what to call this prop. I considered using `fullTextAvailableOnLink`, but that's not really consistent with Cauldron's naming patterns. This is also a pattern that can lead to accessibility issues if not implemented correctly so I'm torn on the naming here.

Reply:
What if we added another comment here on this prop?

```
/**
   * This prop may cause accessibility issues by hiding important content from users.
   * Only use this prop if you have an alternative way to make the full text accessible.
   */
```

Reply:
Ah, that's a good idea!